tests/e2e/steps/steps_def.go

```diff
@@ -800,8 +903,28 @@ func configureAuthorizationProxyServer(cr csmv1.ContainerStorageModule) error {
 		return err
 	}
 
-	fmt.Println("=== Creating Storage ===")
+	fmt.Printf("=== Generating Admin Token ===\n")
+	adminTkn := exec.Command("karavictl",
+		"admin", "token",
+		"--name", "Admin",
+		"--jwt-signing-secret", "secret",
+		"--refresh-token-expiration", fmt.Sprint(30*24*time.Hour),
+		"--access-token-expiration", fmt.Sprint(2*time.Hour),
+	)
+	b, err = adminTkn.CombinedOutput()
+	if err != nil {
+		return fmt.Errorf("failed to create admin token: %v\nErrMessage:\n%s", err, string(b))
+	}
+
+	fmt.Println("=== Copying Admin Token to Tmp File ===\n ")
```

Comment:
Instead of copying, "Writing Admin Token..." makes more sense.

Reply:
updated